docs/install_guides/installing-openvino-raspbian.md

```diff
@@ -75,7 +75,20 @@
 
       sudo -E ./install_dependencies/install_openvino_dependencies.sh
 
-6. For simplicity, it is useful to create a symbolic link as below:
+6. (Optional) Install *numpy* Python Library:
+
+   .. note::
+
+      This step is required only when you decide to use Python API.
+
+   You can use the ``requirements.txt`` file from the ``opt/intel/openvino_2022.3.1/python/python.<x>`` folder:
+
+   .. code-block:: sh
+
+      cd /opt/intel/openvino_2022.3.1
+      pip3 install -r ./python/python3.<x>/requirements.txt
```

Comment:
BTW, in current master we have merged Raspbian and Linux pages. Should we port this change into 2022.3 separately?

Reply:
These are the changes for 22.3, you meant, 2023.0, right?

Reply:
I meant that in master we have merged Raspbian and Linux into a single page, while here we still have 2 pages, while they are quite similar.

Reply:
Sure :) I will port the changes to the single page. Question here, should we use:

`python3 -m pip install -r ./python/python3.<x>/requirements.txt`

or

`pip3 install -r ./python/python3.<x>/requirements.txt`
?

Reply:
`python3 -m pip install -r ./python/python3.<x>/requirements.txt` is preferable. 
But when we migrate those changes with Raspbian page elimination, it become less important :)